static/app/components/core/compactSelect/compactSelect.tsx

```diff
@@ -171,3 +220,15 @@ export function CompactSelect<Value extends SelectKey>({
     </Control>
   );
 }
+
+function shouldVirtualize<Value extends SelectKey>(
+  items: Array<SelectOptionOrSection<Value>>,
+  virtualThreshold = 150
+) {
+  const hasSections = items.some(item => 'options' in item);
```

Comment:
Lets add a comment to the jsdoc threshold comment and explain that we ignore the threshold for sectioned selects

Reply:
aa2f0341